source/tutorials/otu-clustering.rst

```diff
@@ -52,8 +52,8 @@ Dereplicating a ``SampleData[Sequences]`` artifact
 --------------------------------------------------
 
 If you are beginning your analysis with demultiplexed, quality controlled
-sequences, such as those in a `QIIME 1 ``seqs.fna`` file`_ your first step is
-to import that data into a QIIME 1 artifact. The semantic type used here is
+sequences, such as those in a QIIME 1 ``seqs.fna`` file_ your first step
```

Comment:
Sorry, my [first comment](https://github.com/qiime2/docs/pull/330#discussion_r201359710) wasn't very clear --- I think the original intent of this code was for the link to look like this (displaying as HTML for illustration, we would want to use RST in our implementation):

```html
<a href="http://qiime.org/documentation/file_formats.html#post-split-libraries-fasta-file-overview">
    QIIME 1 ``seqs.fna`` file
</a>
```

Reply:
@thermokarst that makes more sense. I will go ahead and make the changes accordingly.